_P248_TempHumidity_AHT1x.ino

```diff
@@ -86,8 +101,10 @@ bool AHT1XReadMesurement(uint8_t address) {
   Wire.requestFrom(address, (uint8_t) 6);
 
   for(uint8_t i = 0; Wire.available() > 0; i++) {
-     AHT10_rawDataBuffer[i] = Wire.read();
+    if (i==6) break;
+    AHT10_rawDataBuffer[i] = Wire.read();
```

Comment:
I would have expected something like this:
`for(uint8_t i = 0; Wire.available() > 0 && i < 6; i++) {`

Your current check comes too late, as i should be in the range 0..5